.github/workflows/release.yml

```
@@ -20,7 +20,7 @@ jobs:
       - name: Setup Node
         uses: actions/setup-node@v3
```

wonder if this version could be bumped to:

```suggestion
        uses: actions/setup-node@v6
```